lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -516,17 +549,16 @@ function checkIp(value) {
         } catch (err) {
             return errString;
         }
-
-        // Apply the existing IP validation logic to each element
-        const validateIpRegex = (ip) => {
-            if (constants.ipv4Regex.test(ip)) {
-                return ip.split('.').every(part => parseInt(part, 10) <= 255);
-            }
-            if (constants.ipv6Regex.test(ip)) {
-                return ip.split(':').every(part => part.length <= 4);
+        */
+        try {
+            parseCIDR(values[i]);
+        } catch (err) {
+            if (!isValid(values[i])) {
+                // eslint-disable-next-line no-console
+                console.log('=== INVALID IP ===', values[i], err);
```

Comment:
Leftover from testing?

Reply:
yep just to print, I should drop the 2 last commits